utils/vulkan-utils.cabal

```diff
@@ -62,6 +63,7 @@ library
     , text
     , transformers
     , typed-process
+    , unordered-containers
```

Comment:
Could you edit the `package.yaml`s instead and run `hpack`? (I already did that here: https://github.com/expipiplus1/vulkan/commit/28c67d900ae0ad8de71ca2f716fc4beff35660fb)